src/components/ebay-carousel/component.js

```diff
@@ -91,14 +91,13 @@ function onRender() {
 
         // Ensure only visible items within the carousel are focusable.
         // We don't have access to these items in the template so me must update manually.
-        forEls(listEl, itemEl => {
```

Comment:
Is there any reason we couldn't move this into `renderFrame` below?

Reply:
This is moved into the render frame. Not sure what you mean by this.

Reply:
Basically I took this code and wrapped it in a `requestAnimationFrame`. I thought this is what the fix should be since it will have 1 request animation frame.

Reply:
I think I see what you mean. I do not think it's a good idea since this is under an if statement to do preserveItems or not. That boolean is flipped in the renderFrame below. A
lso there is a chance that that renderFrame below is not called in certain cases since there is an early return, such as on autoplay. 

I think these things should be revisited when we refactor, otherwise trying to change it like it is now will cause random bugs to creep up.